make-all.bash

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash -e
 
 # Build the syncthing library
-
-./make-go.bash arm
```

Comment:
I still think we should bootstrap latest go as part of the build, as this way we become reliant on Debian to release go fast enough to pick up security fixes and performance improvements.